README.md

```diff
@@ -5,12 +5,13 @@
 
 # Overview
 
-**Flow-Logs Pipeline** (a.k.a. FLP) is an **observability tool** that consumes raw **network flow-logs** in
-their original format 
+**Flow-Logs Pipeline** (a.k.a. FLP) is an **observability tool** that consumes either:
+- raw **network flow-logs** in their original format 
 ([NetFlow v5,v9](https://en.wikipedia.org/wiki/NetFlow) or [IPFIX](https://en.wikipedia.org/wiki/IP_Flow_Information_Export)) 
-and uses a pipe-line to transform the logs into 
-time series metrics in **[prometheus](https://prometheus.io/)** format and in parallel
-transform and persist the logs also into **[loki](https://grafana.com/oss/loki/)**.
+- [eBPF agent](https://github.com/netobserv/netobserv-ebpf-agent) flows in binary format (protobuf+GRPC)
+- Kafka entries in JSON format
```

Comment:
I would split that in different sentences or even paragraphs. Here it makes thinking that the prom/loki capabilities are only available when used with kafka (it's on the same bullet point)

Reply:
also keep in mind that FLP is not limited to its usage from NetObserv : the pipeline is configurable, it can be with or without prom, with or without loki.

Reply:
thanks! lgtm